index.bs

```diff
@@ -120,8 +120,9 @@ Framework {#framework}
   [Exposed=(Window)]
   interface Sanitizer {
     constructor(SanitizerConfig? config);
-    DOMString toString(DOMString input);
-    DocumentFragment toFragment(DOMString input);
+    DocumentFragment sanitize(HTMLString input);
```

Comment:
Is following a better way to represent that _sanitize_ can take a string (if _SanitizerConfig_ is default config or stricter) but may need a _TrustedHTML_ (if _SanitizerConfig_ is weak AND TT is enforced)?
```suggestion
    DocumentFragment sanitize(HTMLString or DOMString input);
```

Reply:
Will take _SanitizerInput_ suggested :)